File: src/utils/novelfull/write.js
const axios = require('axios');
const cheerio = require('cheerio')  ;

const baseUrl = 'https://novelfull.com';
const url = 'latest-release-novel';

const headers = {
    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.82 Safari/537.36',
    'Accept-Language': 'en-US,en;q=0.9,fr;q=0.8',
    'Accept-Encoding': 'gzip, deflate, br',
    'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
    'Connection': 'keep-alive',
};

const getNovelsLatest = async (numberpage) => {
    const { data } = await axios.get(`${baseUrl}/${url}?page=${numberpage}`, { headers });
    const $ = cheerio.load(data);
    const listPage = $('.col-xs-12.col-sm-12.col-md-9.col-truyen-main.archive');
    const rows = listPage.find('.row');
    
    
    const result = [];
    rows.each((i, row) => {
        const image = $(row).find('img').attr('src');
        const title = $(row).find('.col-xs-7 a').text();
        const chapter = $(row).find('.col-xs-2.text-info span').text();
        result.push({ image: baseUrl + image, title, chapter});
    });
    console.log(result[0]);
}

getNovelsLatest(2);

File: src/utils/novelfull/write.py
import sys
import os
from urllib.parse import urlparse
from bs4 import BeautifulSoup
from requests import get
from pathlib import Path
from time import sleep

from bs4 import BeautifulSoup


def GetNovelsLatest(baseUrl,url,numberpage=1):
    header = {'User-Agent': 'Mozilla / 5.0 (Linux ; Android 10 ; SM-G980F Build / QP1A.190711.020 ; wv) AppleWebKit / 537.36 (KHTML, comme Gecko) Version / 4.0 Chrome / 78.0.3904.96 Mobile Safari / 537.36 ',
    "Access-Control-Allow-Origin": "*"}
    page = get(baseUrl +'/'+ url+'?page='+str(numberpage),headers=header).text
    
    soup = BeautifulSoup(page,"html.parser")
    list_page = soup.find('div',class_='col-xs-12 col-sm-12 col-md-9 col-truyen-main archive')
    rows = list_page.find_all('div',{'class':'row'})
    
    data =[]
    
    for row in rows:
        image = row.find('img')['src']
        title = (row.find('div',class_='col-xs-7')).find('a').text
        chapter = (row.find('div',class_='col-xs-2 text-info')).find('span').text
        data.append({'image':baseUrl+image,'title':title,'last-chapter':chapter})
    
    print(data[0])

GetNovelsLatest('https://novelfull.com','latest-release-novel',2)